DIPs/DIP8484.md

```diff
@@ -0,0 +1,95 @@
+# Unleash the traits
```

Comment:
While reading prose is great, maybe you can use a more informative title that tells directly about the purpose of this DIP as often only the title is shown.